rclip/main.py

```diff
@@ -121,10 +121,12 @@ def ensure_index(self, directory: str):
 
     self._db.commit()
 
-  def search(self, query: str, directory: str, top_k: int = 10) -> List[SearchResult]:
+  def search(self, query: str, directory: str, top_k: int = 10,
```

Comment:
Let's move `self` to a new line.

Reply:
I think moving `self` doesn't save enough room to stop pycodestyle from complaining (or maybe I misunderstood the feedback)

Reply:
@ramayer, let's keep it multiline, but move the `self` to the next line. I like when the indent is even :sweat_smile:

e.g.:

```python
def search(self, query: str, directory: str, top_k: int = 10,
                  positive_phrases: List[str] = [], negative_phrases: List[str] = []) -> List[SearchResult]:
```

vs

```python
def search(
    self, query: str, directory: str, top_k: int = 10,
    positive_phrases: List[str] = [], negative_phrases: List[str] = []) -> List[SearchResult]:
```

I think that PEP allows both :smile: 

Sorry for the unclear feedback.